gpcontrib/gp_orca/gporca.c

```diff
@@ -112,6 +137,113 @@ gp_orca_planner(Query *parse, int cursorOptions, ParamListInfo boundParams)
 	return result;
 }
 
+/*
+ * gp_orca_explain_dxl -
+ *	  print out the execution plan for one Query in DXL format
+ *	  this function implicitly uses optimizer
+ */
+static void
+gp_orca_explain_dxl(Query *query, ExplainState *es, const char *queryString,
```

Comment:
Can we do this, since it's not used?
```suggestion
gp_orca_explain_dxl(Query *query, ExplainState *es,
```